Wash history --follow panics when attempting to follow a finished command

```
➜  mnt073479818 ls
aws                    docker                 kubernetes             mock_docker            schema_less_plugin     schema_less_plugin_two
➜  mnt073479818 whistory
1  2019-08-04 08:36  ls -G .
2  2019-08-04 08:36  ls -G
3  2019-08-04 08:37  wash find aws -meta .tags -exists
4  2019-08-04 08:37  wash whistory
5  2019-08-04 08:37  wash whistory --follow 4
6  2019-08-04 08:38  ls -G
7  2019-08-04 08:38  wash whistory
➜  mnt073479818 whistory --follow 6
panic: Unexpected time format

goroutine 1 [running]:
github.com/puppetlabs/wash/cmd.printJournalEntry(0x7ffeefbfecaf, 0x1, 0x1, 0x0, 0x0)
	/Users/enis.inan/GitHub/wash/cmd/history.go:67 +0x7a0
github.com/puppetlabs/wash/cmd.historyMain(0xc0004ba500, 0xc000274b40, 0x1, 0x2, 0x0)
	/Users/enis.inan/GitHub/wash/cmd/history.go:107 +0x96
github.com/puppetlabs/wash/cmd.toRunE.func1(0xc0004ba500, 0xc000274b40, 0x1, 0x2, 0x0, 0x0)
	/Users/enis.inan/GitHub/wash/cmd/root.go:32 +0x4d
github.com/spf13/cobra.(*Command).execute(0xc0004ba500, 0xc000274b20, 0x2, 0x2, 0xc0004ba500, 0xc000274b20)
	/Users/enis.inan/workspace/Go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:762 +0x465
github.com/spf13/cobra.(*Command).ExecuteC(0xc00027ac80, 0x0, 0xc0005d9f88, 0x4007d5f)
	/Users/enis.inan/workspace/Go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/enis.inan/workspace/Go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
github.com/puppetlabs/wash/cmd.Execute(0xc0000b4058)
	/Users/enis.inan/GitHub/wash/cmd/root.go:93 +0x9f
main.main()
	/Users/enis.inan/GitHub/wash/wash.go:13 +0x22
```

--- a/cmd/history.go
+++ b/cmd/history.go
@@ -53,19 +53,25 @@ func printJournalEntry(index string, follow bool) error {
 	// Jun 13 15:44:04.433 stdout: 4096 1559079604 1557434981 1559079604 41ed /lib
 	//                     2597536 1552660099 1552660099 1559079604 81ed /lib/libcrypto.so.1.1
 	scanner := bufio.NewScanner(rdr)
-	var line logFmtLine
 	for scanner.Scan() {
+		var line, empty logFmtLine
 		if err := logfmt.Unmarshal(scanner.Bytes(), &line); err != nil {
 			cmdutil.ErrPrintf("Error parsing %v: %v\n", line, err)
 			continue
 		}
 
-		lines := strings.Split(line.Msg, "\n")
+		if line == empty {
+			// Parser ignored incomplete line rather than erroring. Skip it.
+			continue
+		}
+
 		// TODO: add option to print the original longer time format.
 		t, err := time.Parse(time.RFC3339Nano, line.Time)
 		if err != nil {
 			panic(fmt.Sprintf("Unexpected time format %s", line.Time))
 		}
+
+		lines := strings.Split(line.Msg, "\n")
 		timeStr := t.Format(time.StampMilli)
 		fmt.Println(timeStr, lines[0])
 		if len(lines) > 1 {

